.github/workflows/e2e.yml

```
@@ -58,4 +59,5 @@ jobs:
 
       - name: TestContainer
         run: |
-          mvn -B -T 4 integration-test -pl testcontainer
+#          TODO: add cluster test
+          mvn -B -T 4 integration-test -pl testcontainer -P sync
```

How about using test-sync? And consider changing the first line `name: E2E Tests` to `name: sync Tests`?

```suggestion
          mvn -B -T 4 integration-test -pl testcontainer -P test-sync
```

---

1. rename sync -> test-sync
2. I don't think it's necessary to create a separate test category for sync... What's more, there are still some other tests here(although they are simple).  BTW, are we going to add cluster test in future？

---

IMO, one workflow is for one kind of test. We don’t need to test other modules in sync test at all…

We actually going to abandon the e2e test for cluster since we are refactoring the IT framework for new cluster.